ui/apps/akasha/src/extensions/image-block/image-editor-block.tsx

```diff
@@ -167,10 +162,28 @@ export const ImageEditorBlock = (
     [createBlock, retryCreate],
   );
 
+  const isImgUrl = async url => {
+    try {
+      const response = await fetch(url, { method: 'HEAD' });
+      const isImg = response?.headers?.get('Content-Type')?.startsWith('image');
+      return isImg;
+    } catch (error) {
+      setURLNotImage(true);
+      const notifMsg = error?.message;
+      _uiEvents.current.next({
+        event: NotificationEvents.ShowNotification,
+        data: {
+          type: NotificationTypes.Error,
+          message: notifMsg,
+        },
+      });
+    }
+  };
```

Comment:
```suggestion
  const isImgUrl = async url => {
      const response = await fetch(url, { method: 'HEAD' });
      const isImg = response?.headers?.get('Content-Type')?.startsWith('image');
      return isImg;
  };
```

Reply:
The try catch here is for error on the fetch, which is different from the error that can appear on the upload. Why did you suggest the change?

Reply:
The intent is to make `isImgUrl` focus on a single concern as possible.  And I also noticed `isImgUrl` is only used inside `handleChange` method.